netmiko/yotc/yotc.py

```diff
@@ -78,13 +81,16 @@ def check_config_mode(self, check_string=")#", pattern="#"):
     def save_config(self, cmd="wr", confirm=True, confirm_response="y"):
         """Saves Config."""
         self.exit_config_mode()
-        return super().save_config(
-            cmd=cmd, confirm=confirm, confirm_response=confirm_response
```

Comment:
Is there some reason the `super()` call doesn't work--it should (given the behavior you are doing with `send_command_timing`?

I have a very strong preference for calling the parent class via super() as otherwise the Netmiko code becomes very hard to maintain (i.e. I have to search and edit a large number of files whenever there is a behavior change).

Reply:
if the super.save_config() can add ```normalize=False``` option it can be solved, because when type "y\n",  the device think its not "y", so save_config will canelled

Reply:
Okay, that makes sense...now I understand why it wasn't working in the super-call.